### PR TITLE
cloudsql-ha-dr/readme-Update

### PR DESCRIPTION
**Pull request by gruizmm**

### CODE REVIEW

File: cloudsql-ha-dr/README.md [hunk 1]
```diff
@@ -1,30 +1,49 @@
...
 
 
-# Cloud SQL with High Availability and Disaster Recovery  
+# Host a Highly Available SQL database on Google Cloud
 
-## Description
+## Introduction
 
-This example demonstrates how to deploy a CloudSQL instance with high-availability and cross-region replica.
+This architecture uses click-to-deploy to demonstrate how to deploy a CloudSQL instance with high-availability and cross-region replica.
```
Comment (sylvioneto) [inline cloudsql-ha-dr/README.md:8]:
```suggestion
This architecture uses click-to-deploy to demonstrate how to deploy a Cloud SQL instance with high-availability and cross-region replica.
```

File: cloudsql-ha-dr/README.md [hunk 2]
```diff
@@ -1,30 +1,49 @@
...
 ## Architecture
-Please find below a reference architecture.
-![architecture](architecture.png)
 
+<p align="center"><img src="architecture.png"></p>
+
+The main components that we would be setting up are (to learn more about these products, click on the hyperlinks).
+
+* [VPC](https://cloud.google.com/vpc) : Global virtual network that spans all regions. Single VPC for an entire organization, isolated within projects. Increase IP space with no downtime.
+* CloudSQL for Postgres instance with [high-availability](https://cloud.google.com/sql/docs/postgres/high-availability) : The purpose of an HA configuration is to reduce downtime when a zone or instance becomes unavailable.
```
Comment (sylvioneto) [inline cloudsql-ha-dr/README.md:35]:
```suggestion
* Cloud SQL for Postgres instance with [high-availability](https://cloud.google.com/sql/docs/postgres/high-availability) : The purpose of an HA configuration is to reduce downtime when a zone or instance becomes unavailable. 
```

File: cloudsql-ha-dr/README.md [hunk 3]
```diff
@@ -1,30 +1,49 @@
...
-Please find below a reference architecture.
-![architecture](architecture.png)
 
+<p align="center"><img src="architecture.png"></p>
+
+The main components that we would be setting up are (to learn more about these products, click on the hyperlinks).
+
+* [VPC](https://cloud.google.com/vpc) : Global virtual network that spans all regions. Single VPC for an entire organization, isolated within projects. Increase IP space with no downtime.
+* CloudSQL for Postgres instance with [high-availability](https://cloud.google.com/sql/docs/postgres/high-availability) : The purpose of an HA configuration is to reduce downtime when a zone or instance becomes unavailable. 
+* CloudSQL [cross-region read replica](https://cloud.google.com/sql/docs/postgres/intro-to-cloud-sql-disaster-recovery) instance : database disaster recovery (DR) is about providing continuity of processing, specifically when a region fails or becomes unavailable.
```
Comment (sylvioneto) [inline cloudsql-ha-dr/README.md:36]:
```suggestion
* Cloud SQL [cross-region read replica](https://cloud.google.com/sql/docs/postgres/intro-to-cloud-sql-disaster-recovery) instance : database disaster recovery (DR) is about providing continuity of processing, specifically when a region fails or becomes unavailable.
```

File: cloudsql-ha-dr/README.md [hunk 4]
```diff
@@ -1,30 +1,49 @@
...
+By implementing this solution, organizations gain peace of mind knowing that their critical databases are protected and accessible, even during unforeseen events or outages. This solution ensures business continuity by minimizing downtime, providing robust data protection through automated backups, and enabling seamless failover and disaster recovery processes.
 
-:clock1: Estimated deployment time: 16 min 30 sec
+This solution is suitable for a wide range of applications and industries, including e-commerce platforms, financial systems, customer relationship management (CRM) tools, and more.
+
+## Use cases
+
+These are some examples of the use cases you can build on top of this architecture:
 
-:heavy_dollar_sign: Estimated solution cost: [USD 706.44 per 1 month](https://cloud.google.com/products/calculator/#id=358b4a68-4f82-4e88-84f4-1da99a05548b)
```
Comment (sylvioneto) [inline cloudsql-ha-dr/README.md:18]: I see you removed the estimation here but did not add it to the costs section. Could you please take a look?

**Review by sylvioneto — CHANGES_REQUESTED** [review]
Thank you @gruizmm for your PR. I've added some suggestions and requested few changes.

File: cloudsql-ha-dr/README.md (hunk 4 above)
Comment (gruizmm) [reply cloudsql-ha-dr/README.md:18]: I actually moved this to the cost section under the architecture, the text changed but the link to the calculator is the same.

**Review by sylvioneto — APPROVED** [review]
lgtm

**Pull request merged**